build.cake

```diff
@@ -49,13 +55,55 @@ Task("Clean")
 {
     CleanDirectories("./**/obj", (fsi)=> !fsi.Path.FullPath.Contains("XFCorePostProcessor") && !fsi.Path.FullPath.StartsWith("tools"));
     CleanDirectories("./**/bin", (fsi)=> !fsi.Path.FullPath.Contains("XFCorePostProcessor") && !fsi.Path.FullPath.StartsWith("tools"));
-
 });
 
 Task("NuGetPack")
     .IsDependentOn("Build")
     .IsDependentOn("_NuGetPack");
 
+Task("provision")
+    .Does(async () =>
+    {
+        Information ("ANDROID_HOME: {0}", ANDROID_HOME);
+
+        var androidSdkSettings = new AndroidSdkManagerToolSettings { 
+            SdkRoot = ANDROID_HOME,
+            SkipVersionCheck = true
+        };
+
+        try { AcceptLicenses (androidSdkSettings); } catch { }
+
+        AndroidSdkManagerInstall (new [] {
+                "platforms;android-29"
+            }, androidSdkSettings);
+
+        var platform = IsRunningOnWindows() ? "windows" : "macos";
+        await Boots ($"https://aka.ms/xamarin-android-commercial-d16-2-{platform}");
+
+        string monoVersion = "6.0.0.319";
```

Comment:
can't we keep the monoVersion as a variable at the top-level ?

Reply:
https://github.com/xamarin/Xamarin.Forms/pull/7458/files#diff-4d3634747586b964df9f7712bc3ec669R49